### DISCUSSION TITLE
Allow loading multiple skins at once

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#19895] [Ideas]
Instead of having a single skin, a nice feature would be to have a tiered skin system similar to minecraft. Skins higher on the tiered list would be chosen first, and skins below would fill any gaps where there are no assets. This would allow for changing hit circles without changing anything else without having to modify each skin individually.
The system could work by CTRL clicking to add a skin to the bottom of the list (a number in brackets would show where the skin is in the list) and SHIFT clicking to remove a skin from the list.

**Comment by peppy** [comment]
Already implemented that in the past, but it was too complicated. We'll add the ability to copy skin content to another skin but probably not "skin layers".
